src/routes/Home.tsx

```diff
@@ -1,8 +1,17 @@
+import { useState } from "react";
+import Carousel from "../components/Slider";
+import carouselData from "../../mock-data/SliderItems";
```

Comment:
carouselData???????
are you fuckin serius?

Reply:
easyyyy!!!😒